mirror-nixos-branch.pl

```diff
@@ -210,27 +210,35 @@ sub fetch {
     }
 
     if ($channelName =~ /nixos/) {
+        my $arch = "x86_64-linux";
+        if ($channelName =~ /-aarch64/) {
+            $arch = "aarch64-linux";
+        }
+
         downloadFile("nixos.channel", "nixexprs.tar.xz");
-        downloadFile("nixos.iso_minimal.x86_64-linux");
+        downloadFile("nixos.iso_minimal.$arch");
         downloadFile("nixpkgs.tarball", "packages.json.br", "json-br");
-        downloadFile("nixos.options", "options.json.br", "json-br");
 
-        if ($channelName !~ /-small/) {
-            downloadFile("nixos.iso_minimal.i686-linux");
+        # Only x86-specific
```

Comment:
```suggestion
        # Only built on the main channel (x86_64-linux)
```

It's not really that it's specific to x86, but that other channels for NixOS don'e have it.